lib/rules/jsx-no-bind.js

```diff
@@ -5,6 +5,8 @@
  */
 'use strict';
 
+var propName = require('jsx-ast-utils').propName;
```

Comment:
it'd be nice if we could require via path, ie, `var propName = require('jsx-ast-utils/propName')` rather than having to pull everything in and grab one property - but obv this is not a blocker :-)